ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -506,6 +506,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-aws-ipi-disconnected-private-tp-p3-long-duration-part2-3
```

Comment:
Look good to me, can we please name the job as `e2e-aws-ipi-disconnected-private-tp-p3-long-duration-cloud-partx-x`

Reply:
cc @liangxia

Reply:
Could you sort them in alphanumeric order ?

Reply:
Done

Reply:
BTW, if we decided to add `cloud` in the name, perhaps we do not need `partx-y`.  Since `partx-y` means to add feature1/feature2 in part1, and feature3/feature4 in part2, ... for the whole team.
We only need one of them (cloud or partx-y) IMO.

Reply:
@liangxia I have updated based on your comment, PTAL